.gitignore

```diff
@@ -1,37 +1,92 @@
-# Logs
-logs
-*.log
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
 # Coverage directory used by tools like istanbul
 coverage
 
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# node-waf configuration
-.lock-wscript
-
-# Compiled binary addons (http://nodejs.org/api/addons.html)
-build/Release
-
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
 
-# Vim
-*.swp
-
-# Generated JavaScript
-/test/browserified.js
-/browserified.js
+# Build artifacts
+/es/h.d.ts
+/es/h.js
+/es/h.js.map
+/es/helpers/attachto.d.ts
+/es/helpers/attachto.js
+/es/helpers/attachto.js.map
+/es/hooks.d.ts
+/es/hooks.js
+/es/hooks.js.map
+/es/htmldomapi.d.ts
+/es/htmldomapi.js
+/es/htmldomapi.js.map
+/es/is.d.ts
+/es/is.js
+/es/is.js.map
+/es/modules/attributes.d.ts
+/es/modules/attributes.js
+/es/modules/attributes.js.map
+/es/modules/class.d.ts
+/es/modules/class.js
+/es/modules/class.js.map
+/es/modules/dataset.d.ts
+/es/modules/dataset.js
+/es/modules/dataset.js.map
+/es/modules/eventlisteners.d.ts
+/es/modules/eventlisteners.js
+/es/modules/eventlisteners.js.map
+/es/modules/hero.d.ts
+/es/modules/hero.js
+/es/modules/hero.js.map
+/es/modules/module.d.ts
+/es/modules/module.js
+/es/modules/module.js.map
+/es/modules/props.d.ts
+/es/modules/props.js
+/es/modules/props.js.map
+/es/modules/style.d.ts
+/es/modules/style.js
+/es/modules/style.js.map
+/es/snabbdom.bundle.d.ts
+/es/snabbdom.bundle.js
+/es/snabbdom.bundle.js.map
+/es/snabbdom.d.ts
+/es/snabbdom.js
+/es/snabbdom.js.map
+/es/thunk.d.ts
+/es/thunk.js
+/es/thunk.js.map
+/es/tovnode.d.ts
+/es/tovnode.js
+/es/tovnode.js.map
+/es/vnode.d.ts
+/es/vnode.js
+/es/vnode.js.map
+/modules/attributes.d.ts
+/modules/attributes.js
+/modules/attributes.js.map
+/modules/class.d.ts
+/modules/class.js
+/modules/class.js.map
+/modules/dataset.d.ts
+/modules/dataset.js
+/modules/dataset.js.map
+/modules/eventlisteners.d.ts
+/modules/eventlisteners.js
+/modules/eventlisteners.js.map
+/modules/hero.d.ts
+/modules/hero.js
+/modules/hero.js.map
+/modules/module.d.ts
+/modules/module.js
+/modules/module.js.map
+/modules/props.d.ts
+/modules/props.js
+/modules/props.js.map
+/modules/style.d.ts
+/modules/style.js
+/modules/style.js.map
+/helpers/attachto.d.ts
+/helpers/attachto.js
+/helpers/attachto.js.map
```

Comment:
What I did here is expanded the

```
/modules
/helpers
/es
```

into an explicit and complete list of files.

In most of my projects, I do ignore whole directories, but in those cases I build everything into one or more target directories. In this project, we have a rather unfortunate setup where we build some files into the root directory and others into sub-directories.

What I do here is use git's ignore mechanism to help me identify and list the files for inclusion in the package. I know that if this list is explicit and `git status` shows that my directory is clean, then I can use this same list as the value for `files` in `package.json`. Of course, I did check for extraneous entries. And that's what I did. I copy-pasted this list into `package.json` for the `files` field value.